android/build.gradle

```diff
@@ -29,7 +29,7 @@ android {
         minSdkVersion getExtOrIntegerDefault('minSdkVersion')
         targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
         versionCode 1
-        versionName '4.6.1'
+        versionName '5.0.0'
```

Comment:
Should this be `5.0.0-rc.1` instead? Honestly don't know how gradle works.

Reply:
I think that's fine?